superset/config.py

```diff
@@ -369,6 +370,7 @@ def _try_json_readsha(  # pylint: disable=unused-argument
 CACHE_DEFAULT_TIMEOUT = 60 * 60 * 24
 CACHE_CONFIG: CacheConfig = {"CACHE_TYPE": "null"}
 TABLE_NAMES_CACHE_CONFIG: CacheConfig = {"CACHE_TYPE": "null"}
+DASHBOARD_CACHE_TIMEOUT = 60 * 60 * 24 * 14
```

Comment:
I do not like this assumption. I didn't see any typical pattern for dashboard updates. 
Why do we need this cache_timeout for dashboard?

Reply:
I just feel one day is too short. Normally when a new dashboard is published and PSA sent, users may still trickle in after one or two weeks.

Since this cache is actively purged when things are updating, it's OK to set a longer expiration time.

Reply:
I think Grace's point is to give this an infinite timeout, since it'll be purged. i agree with that

Reply:
I checked the data and there indeed are some popular dashboards that haven't been updated for almost half a year. I'll set the expiration date to 1 year.